meeshkan/serve/mock/faker/faker_exception.py

```diff
@@ -0,0 +1,10 @@
+
+class FakerException(Exception):
+    def __init__(self, *args):
```

Comment:
I wonder if the implementation here provides any extra value over something a bit simpler like
```python
class FakerException(Exception)
  """Something went wrong when generating mock data for request."""
```
?